### ISSUE TITLE
Emoji Panel not working after fresh Win10 Install with MicroWin

### ISSUE THREAD

**Issue by Flammodo** [ChrisTitusTech/winutil#3030]
## Describe the bug
The Shortcut Win+. does not work anymore on fresh win10 install. It simply does not open when using the shortcut. I noticed that the Onscreen Keyboard Button in tray does't work also. It won't open the Keyboard on click. Tried some tips from the internet but none of these fixed that issue. Does MircoWin disable the Onscreen Keyboard Service or something similar? How can I get it working again?
Because it might be important: My system language ist german.

## To Reproduce
1. Press 'Win+.'

**Comment by CodingWonders** [comment]
It works for me:

![vmware_HfribJwcRE](https://github.com/user-attachments/assets/c8346a2a-c28d-4758-ac9b-c0e01aa326fb)

**Comment by zmileski** [comment]
Everything working fine...
![t  keyborad](https://github.com/user-attachments/assets/32017153-0e2f-4676-9fc4-6712513cf84a)

![emoji](https://github.com/user-attachments/assets/75a579b1-a66f-40c9-99eb-69619b974a88)
![keyborad on screen](https://github.com/user-attachments/assets/5441bbe7-60f1-4e13-8e99-fd44fa23875f)

**Comment by Code-Leaf-Crash** [comment]
Same issue here. However I can open the onscreen keyboard but not the emoji panel. Also, might be unrelated but my spellcheck doesn't work and I don't get any word suggestions when typing after installing windows with MicroWin.

**Comment by zmileski** [comment]
If you using old windows build try change keyboard language to en_US or just update the windows

**Comment by CodingWonders** [comment]
@Code-Leaf-Crash, what Windows 10 build are you using? Go to `winver`

**Comment by Code-Leaf-Crash** [comment]
> @Code-Leaf-Crash, what Windows 10 build are you using? Go to `winver`

"Version 22H2 (OS Build 19045.5073)"  :) 

I tried to change my keyboard to en_US but that didn't do anything either unfortunately. I also don't have any updates in windows update. Hope that helps :)

**Comment by Flammodo** [comment]
I figured out how to solve this issue: I tried many things but only reregistering the Win Components helped.
Just running Get-AppxPackage -AllUsers| Foreach {Add-AppxPackage -DisableDevelopmentMode -Register "$($_.InstallLocation)\AppXManifest.xml"} in PowerShell.
That repaired the panel for me (and also the Snipping Tool Shortcut via print).
But it is still not clear for me why this happened after a fresh win install.

Hope that helps someone else also!
